openapi.yaml

```diff
@@ -3565,7 +3573,12 @@ paths:
                           to the user.
                         example: 1073741824
                   budget:
-                    $ref: '#/components/schemas/budget'
+                    type: number
+                    nullable: true
+                    description: |-
+                      The remaining budget a user has available.
+                      The value MUST be specified in the currency of the back-end.
+                      The value SHOULD be set to `null` if no explicit limit applies.
```

Comment:
> SHOULD be set to `null` if no explicit limit applies.

is that equivalent to omitting the field?

Reply:
If it's not set, the meaning is pretty much undefined right now.